draft/spec/index.html

```diff
@@ -680,9 +680,10 @@ <h2>Manifest</h2>
                 <span id="E107">MUST</span> correspond to a digest value key found in one or more <code>state</code>
                 blocks of the current and/or previous <code>version</code> blocks of the <a>OCFL Object</a>. The
                 value for each key <span id="E092">MUST</span> be an array containing the <a>content path</a>s of files
-                in the OCFL Object that have content with the given digest. As JSON keys are case sensitive, while
-                digests may not be, there is an additional requirement that each digest value
-                <span id="E096">MUST</span> occur only once in the manifest regardless of case. Content paths within a
+                in the OCFL Object that have content with the given digest. As JSON keys are case sensitive,
+                for digest algorithms with case insensitive digest values, there is an additional requirement that each
```

Comment:
> digest algorithms with case insensitive digest values

Isn't the output of most digest algorithms simply a byte array? Isn't it the encoding of the digest that's of interest here?

Reply:
I think this would be splitting a hair that we have preemptively attempted to glue together by defining the algorithm as the combination of basic digest and encoding, per https://ocfl.io/draft/spec/#digests